proxy/http/HttpSM.cc

```diff
@@ -4622,16 +4623,16 @@ HttpSM::do_http_server_open(bool raw)
   ink_assert(pending_action == NULL);
 
   if (false == t_state.api_server_addr_set) {
-    ink_assert(t_state.current.server->port > 0);
-    t_state.current.server->addr.port() = htons(t_state.current.server->port);
+    ink_assert(t_state.current.server->dst_addr.port() > 0);
+    t_state.current.server->dst_addr.port() = htons(t_state.current.server->dst_addr.port());
```

Comment:
here is where we used to set the port in the addr based on the port uint_16t. not sure this makes sense anymore.